engine/src/test/java/com/datatorrent/stram/StramMiniClusterTest.java

```diff
@@ -125,19 +124,10 @@ public static void setup() throws InterruptedException, IOException
     conf = StramClientUtils.addDTDefaultResources(conf);
     conf.setInt(YarnConfiguration.RM_SCHEDULER_MINIMUM_ALLOCATION_MB, 64);
     conf.setInt("yarn.nodemanager.vmem-pmem-ratio", 20); // workaround to avoid containers being killed because java allocated too much vmem
-    conf.setStrings("yarn.scheduler.capacity.root.queues", "default");
-    conf.setStrings("yarn.scheduler.capacity.root.default.capacity", "100");
-
-    StringBuilder adminEnv = new StringBuilder(1024);
-    if (System.getenv("JAVA_HOME") == null) {
-      adminEnv.append("JAVA_HOME=").append(System.getProperty("java.home"));
-      adminEnv.append(",");
-    }
-    adminEnv.append("MALLOC_ARENA_MAX=4"); // see MAPREDUCE-3068, MAPREDUCE-3065
-    adminEnv.append(",");
-    adminEnv.append("CLASSPATH=").append(getTestRuntimeClasspath());
-
-    conf.set(YarnConfiguration.NM_ADMIN_USER_ENV, adminEnv.toString());
+    conf.set("yarn.scheduler.capacity.root.queues", "default");
+    conf.set("yarn.scheduler.capacity.root.default.capacity", "100");
+    conf.set(YarnConfiguration.NM_ADMIN_USER_ENV, String.format("JAVA_HOME=%s,CLASSPATH=%s", System.getProperty("java.home"), getTestRuntimeClasspath()));
```

Comment:
What about MALLOC_ARENA_MAX?

Reply:
Not needed, it is not a production cluster.

Reply:
Can you add that comment to the commit message so we know why the change was made.

Reply:
Why? There was no reason to have that setting for the YarnMiniCluster in the first place.

Reply:
In case if somebody is looking back in the future why the line was removed, the commit message will have the reason. Just for clarity not asking for a comment in code, just adding something to the commit message that along with the jvm change it also removes this property that it is not needed.

Reply:
I can add this to JIRA, will it be sufficient?

Reply:
It would be better to add at least something general to the commit message like removed unused configuration properties. If you are not inclined to it I would ok in this case as it is something that was not in effect.

Reply:
I prefer JIRA over a comment or a commit message.